packages/cli/config/eslint-factory.js

```diff
@@ -84,6 +84,7 @@ function createConfig(dir, extraConfig = {}) {
       ...(ignorePatterns ?? []),
     ],
     rules: {
+      'deprecation/deprecation': 'off',
```

Comment:
If I understand this correctly this would turn of the deprecation warnings for everyone using the `eslint-factory` - so all linting in plugins & apps would no longer respect `deprecation` warnings by default. Is this really what you aim for?